<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoMagDialog</class>
 <widget class="QDialog" name="GeoMagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geomagnetic Declination</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>328</width>
        <height>553</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="decLocGrpBx">
         <property name="title">
          <string>Location</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="1" column="1" rowspan="2">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="margin">
              <number>6</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="decXDblSpinBx">
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="decYDblSpinBx">
                 <property name="decimals">
                  <number>4</number>
                 </property>
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Capture from map</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="captureLocBtn">
                 <property name="minimumSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>26</width>
                   <height>26</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/plugins/fossildigtools/icons/capturepin-origin.svg</normaloff>:/plugins/fossildigtools/icons/capturepin-origin.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
            <zorder></zorder>
            <zorder></zorder>
            <zorder></zorder>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="pinOriginLabel">
            <property name="text">
             <string>Origin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="decOriginLineEdit">
            <property name="styleSheet">
             <string notr="true">QLineEdit {background-color: rgba(255, 255, 255, 128);}</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Coords</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Elevation</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QDoubleSpinBox" name="decElevDblSpnBx">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>30000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="decElevUnitsCmbBx">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>ft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>m</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="decDeclinationGrpBx">
         <property name="title">
          <string>Declination</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Magnetic bearings to &lt;span style=&quot; font-weight:600; color:#4243fd;&quot;&gt;true&lt;/span&gt; directions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="decCurrentDblSpinBox">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Current (off true North)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>6</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <property name="horizontalSpacing">
                <number>3</number>
               </property>
               <property name="verticalSpacing">
                <number>0</number>
               </property>
               <item row="2" column="2">
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="spacing">
                  <number>3</number>
                 </property>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/plugins/fossildigtools/icons/directional.svg</pixmap>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/plugins/fossildigtools/icons/directional.svg</pixmap>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/plugins/fossildigtools/icons/directional.svg</pixmap>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="fromOriginIconLabel">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/plugins/fossildigtools/icons/origin.svg</pixmap>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/plugins/fossildigtools/icons/directional.svg</pixmap>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="4">
                <widget class="QDoubleSpinBox" name="decEBearingDblSpnBx">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>˚</string>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLabel" name="label_18">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>E</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QDoubleSpinBox" name="decSBearingDblSpnBx">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>˚</string>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>S</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QDoubleSpinBox" name="decNBearingDblSpnBx">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>˚</string>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QDoubleSpinBox" name="decWBearingDblSpnBx">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>˚</string>
                 </property>
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>360.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="label_20">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>W</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>N</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_21">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {font-size: 0.75em;}</string>
     </property>
     <property name="text">
      <string>Note: map projection and coordinates must be in UTM (meters)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>decXDblSpinBx</tabstop>
  <tabstop>decYDblSpinBx</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
